src/components/Contacts/ContactDetails/ContactDetailsTab/ContactDetailsTab.tsx

```diff
@@ -65,89 +118,189 @@ export const ContactDetailsTab: React.FC<ContactDetailTabProps> = ({
-          <ContactDetailHeadingText variant="h6">
-            {t('Other')}
-          </ContactDetailHeadingText>
-          <ContactDetailHeadingIcon />
+          ) : (
+            <ContactTags
+              contactId={data.contact.id}
+              contactTags={data.contact.tagList}
+            />
+          )}
         </ContactDetailHeadingContainer>
-        {loading || !data ? (
-          <>
-            <ContactDetailLoadingPlaceHolder variant="rect" />
-            <ContactDetailLoadingPlaceHolder variant="rect" />
-            <ContactDetailLoadingPlaceHolder variant="rect" />
-          </>
-        ) : (
-          <ContactDetailsOther contact={data.contact} />
-        )}
-      </ContactDetailSectionContainer>
-      <Divider />
-      <ContactDeleteButton variant="outlined" color="default">
-        {t('delete contact')}
-      </ContactDeleteButton>
-    </ContactDetailsTabContainer>
+        <Divider />
+        {
+          // People Section
+        }
+        <ContactDetailSectionContainer>
+          <ContactDetailHeadingContainer>
+            <ContactDetailHeadingText variant="h6">
+              {loading || !data ? t('Loading') : data.contact.name}
+            </ContactDetailHeadingText>
+            <ContactDetailHeadingIcon />
+          </ContactDetailHeadingContainer>
+          {loading || !data ? (
+            <>
+              <ContactDetailLoadingPlaceHolder variant="rect" />
+              <ContactDetailLoadingPlaceHolder variant="rect" />
+              <ContactDetailLoadingPlaceHolder variant="rect" />
+            </>
+          ) : (
+            <ContactDetailsTabPeople data={data?.contact} />
+          )}
+        </ContactDetailSectionContainer>
+        <Divider />
+        {
+          // Mailing Section
+        }
+        <ContactDetailSectionContainer>
+          <ContactDetailHeadingContainer>
+            <ContactDetailHeadingText variant="h6">
+              {t('Mailing')}
+            </ContactDetailHeadingText>
+            <ContactDetailHeadingIcon />
+          </ContactDetailHeadingContainer>
+          {loading || !data ? (
+            <>
+              <ContactDetailLoadingPlaceHolder variant="rect" />
+              <ContactDetailLoadingPlaceHolder variant="rect" />
+              <ContactDetailLoadingPlaceHolder variant="rect" />
+            </>
+          ) : (
+            <ContactDetailsTabMailing data={data.contact} />
+          )}
+        </ContactDetailSectionContainer>
+        <Divider />
+        {
+          // other Section
+        }
+        <ContactDetailSectionContainer>
+          <ContactDetailHeadingContainer>
+            <ContactDetailHeadingText variant="h6">
+              {t('Other')}
+            </ContactDetailHeadingText>
+            <ContactDetailHeadingIcon />
+          </ContactDetailHeadingContainer>
+          {loading || !data ? (
+            <>
+              <ContactDetailLoadingPlaceHolder variant="rect" />
```

Comment:
So MPDx is using the material ui lab? Do you find any of the other libraries useful besides the skeleton? I've been thinking of adding it for the skeleton at least.

Reply:
We've used ```Autocomplete``` in a few spots. I haven't personally worked with it yet, but I have a ticket to add it to the contact tags input. The Skeleton component is really nice while waiting for data to load.